web/src/beta/lib/core/Map/hooks.ts

```diff
@@ -112,6 +116,13 @@ export default function ({
     );
   }, [onLayerSelect, selectedLayer]);
 
+  useEffect(() => {
+    if (zoomedLayerId) {
+      engineRef.current?.lookAtLayer(zoomedLayerId);
+      onZoomToLayer?.(undefined);
+    }
+  }, [zoomedLayerId, onZoomToLayer]);
```

Comment:
This is alreadying happening in core/Visualizer/hooks.ts. Any reason it needs to happen in Map too?

Reply:
removed, sorry I forget to clean it

Reply:
Please remove zoomLayerId and onZoomToLayer from the prop type too 🙏🏻